docs/release-notes/3486-use-model-name-as-identifier.txt

```diff
@@ -0,0 +1,20 @@
+:orphan:
+
+**Breaking Changes**
+
+-  API: Routes with `/api/v1/models/:id/*` are replaced by `/api/v1/models/:name/*`. Spaces and
+   special characters in a name must be URI-encoded. You can get a model by ID with
+   `/api/v1/models?id=`.
+
+-  API: On the list of models (`/api/v1/models`) the optional name parameter is now a case-sensitive
+   match, unless you add the parameter `name_case_insensitive=true`.
+
+-  Client: `get_model(int: id)` is deprecated - use `get_model(name: string)` or
+   `get_model_by_id(int: id)`.
+
+-  Client: `get_model_by_name` is deprecated - use `get_model(name: string)`.
```

Comment:
to me, "deprecated" means its use is discouraged but it's still available. maybe this should read "removed" instead?

Reply:
good idea, will do

Reply:
wait, was `get_model_by_name()` released?  I thought it never made it into a release (in which case it doesn't need a release note?)

Reply:
I think you're right `get_model_by_name()` was 6 days ago, it's not in the release but it is [in determined-ee](https://github.com/determined-ai/determined-ee/commit/057a25773e1c7976af2df330980dd419cdfe71d1#diff-2c4cec18a64aa21942e745435466ff4499ee1e9e797990a7ade4461d688a8869) -- I'll remove it from the release notes on Tuesday